apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Resolvers/EsettExchangeResolvers.cs

```diff
@@ -75,4 +75,24 @@ public class EsettExchangeResolvers
                "ResponseDocument",
                _controllerName,
                new { documentId = result.DocumentId });
+
+    public string? GetStorageDocument(
+       [Parent] MeteringGridAreaImbalanceSearchResult result,
+       [Service] IHttpContextAccessor httpContextAccessor,
+       [Service] LinkGenerator linkGenerator) =>
+           linkGenerator.GetUriByAction(
+               httpContextAccessor.HttpContext!,
+               "StorageDocument",
+               _controllerName,
+               new { documentId = result.Id });
```

Comment:
nitpick: can see that this is mostly relevant in the next PR. Hopefully can Git figure out to merge the two without conflicts.